src/main/webapp/app/entities/search/search-modal.tsx

```diff
@@ -192,6 +161,9 @@ const mapStateToProps = (storeState: IRootState) => ({
   showResults: storeState.search.showResults,
```

Comment:
renamed it to searchedResults

Reply:
Flag showResults is use to display visulization UI when click on search button, so showResults name is okay

Reply:
showResults name shows you are hiding or showing results.however  searchedResults shows whatever we have searched. the outcome of search is going to be searched results